packages/fluentui/react-northstar/src/themes/teams/components/Card/cardStyles.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { CardVariables } from './cardVariables';
+import { CardStylesProps } from '../../../../components/Card/Card';
+import { pxToRem } from '../../../../utils';
+
+const cardStyles: ComponentSlotStylesPrepared<CardStylesProps, CardVariables> = {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    return {
+      display: 'flex',
+      flexDirection: 'column',
+      position: 'relative',
+      padding: v.cardPadding,
```

Comment:
```suggestion
      padding: v.padding,
```

We don't use prefixes for roots